README.md

```diff
@@ -315,19 +314,35 @@ Modifiers are subclasses, or variants, of a component. They add a suffix to the
 
 When extending a component and styling the inner elements, use the base component's inner elements' class name for styling, instead of extending the class names of the inner elements as well.
 
+
 ```css
 
 /* Good - modifiers (subclasses) of components refer to unmodified inner element's names */
-.c-list-inline--boxy > .o-list-inline__list-item {}
+.c-assignment-list--boxy > .c-assignment-list__item {}
 
 /* Bad - don't modify inner element's names */
-.c-list-inline--boxy > .c-list-inline__list-item--boxy {}
+.c-assignment-list--boxy > .c-assignment-list__list-item--boxy {}
 
 /* Bad - don't create child elements with the modifier in the name. */
-.c-list-inline--boxy__list-item {}
+.c-assignment-list--boxy__list-item {}
 ```
 
 
+<a name="feature-specific-styles"></a>
+### Feature-specific styles
```

Comment:
I had to make some major changes, due to all the redundancy and some weird ordering. There is a branch w/PR off of this one with the restructuring. I am going to create ANOTHER branch off of THAT one with the remaining fixes listed here:

* Add note about namespacing features with a `ns-{feature}` namespace.
* Differentiate between alpha and v1 vis-a-vis how to include feature styles.

So if you could kindly approve this one, I'll move on to the other two ...